infra/package.json

```
@@ -7,7 +7,7 @@
     "charts": "node -r esbuild-register src/cli/generate-chart-values.ts",
     "update": "yarn update:packagejson",
     "update:packagejson": "node -r esbuild-register scripts/update-package-json.ts",
-    "cli": "node -r esbuild-register src/cli/cli.ts"
+    "infra": "node -r esbuild-register src/cli/cli.ts"
```

Please add a `cli` script target, informing of changed CLI invocation
```suggestion
    "cli": "echo 'Please use `yarn infra` instead of `yarn cli`' && exit 1",
    "infra": "node -r esbuild-register src/cli/cli.ts"
```